src/rest-server/src/models/job.js

```diff
@@ -61,7 +61,7 @@ class Job {
       case 'FRAMEWORK_COMPLETED':
         if (exitCode === 0) {
           jobState = 'SUCCEEDED';
-        } else if (exitCode === 214) {
+        } else if (exitCode === -7352) {
```

Comment:
Can we use const var instead of hard code?

Reply:
Agree, but the code is not owned by me. I just keep no breaking change.
@sunqinzheng to refine this later.